matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/accountdata/UserAccountDataTypes.kt

```diff
@@ -27,4 +27,5 @@ object UserAccountDataTypes {
     const val TYPE_ALLOWED_WIDGETS = "im.vector.setting.allowed_widgets"
     const val TYPE_IDENTITY_SERVER = "m.identity_server"
     const val TYPE_ACCEPTED_TERMS = "m.accepted_terms"
+    const val TYPE_OVERRIDE_COLORS = "im.vector.setting.override_colors"
```

Comment:
does this type need to be added to the matrix spec somewhere?

Reply:
I have another PR that implements the same functionality in element-desktop (https://github.com/matrix-org/matrix-react-sdk/pull/5626).  As far as I understand keys under `im.vector.setting` are free-to-use by clients, so I don't think it's absolutely necessary to include it in the matrix spec (apart from maybe documenting its existence so that other clients can pick this up if they wish)